### PR TITLE
Review

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: lib/format.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const currentDate = {
+  Y: new Date().getFullYear(),
+  D: new Date().getDate(),
+  M: new Date().getMonth() + 1,
+  h: new Date().getHours(),
+  m: new Date().getMinutes(),
+  s: new Date().getSeconds(),
+};
```
Comment (tshemsedinov) [inline lib/format.js:10]:
```suggestion
const now = new Date();
const currentDate = {
  Y: now.getFullYear(),
  D: now.getDate(),
  M: now.getMonth() + 1,
  h: now.getHours(),
  m: now.getMinutes(),
  s: now.getSeconds(),
};
```

File: lib/format.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  const used = [];
+  for (let i = 0; i < date.length; i++) {
+    if (!used.includes(date[i])) {
+      Object.keys(currentDate).map(key => {
+        if (date[i] === key) {
+          result.push(currentDate[key]);
+          used.push(date[i]);
+        }
+      });
+      if (!Object.keys(currentDate).includes(date[i])) result.push(date[i]);
```
Comment (tshemsedinov) [inline lib/format.js:23]: Second appearance of `Object.keys(currentDate)`, need variable

Comment (tshemsedinov) [reply lib/format.js:23]: for `date[i]` as well

**Comment by lightballer** [comment]
Thanks, already add mentioned improvements, will update README.MD soon

**Pull request closed without merging**